.env

```diff
@@ -0,0 +1,2 @@
+SUPABASE_URL = paste here your supabase url
+SUPABASE_KEY = paste here your supabase key
```

Comment:
Please add `.env` to `.gitignore`, and create `.env.example` with:
```
SUPABASE_URL
SUPABASE_KEY
```

Reply:
done